README.md

```diff
@@ -77,12 +77,24 @@ These examples show how you could embed the MSA viewer into your page.
 
 ## Use the MSA viewer
 
+**The following examples assume that the ```msa()``` constructor is available.**
+
+If you have loaded ```msa``` as a script in your web page with something like...
+```
+<script src="//cdn.bio.sh/msa/latest/msa.min.gz.js"></script>
+```
+... then congratulations! You are ready to go.
+
+If you are using ```require``` to import msa into your own library, then you'll need use the following:
```

Comment:
maybe add sth. like "If you are using npm and installed the MSA Viewer as dependency"?